variables.tf

```diff
@@ -31,7 +20,18 @@ variable "members" {
 }
 
 variable "role" {
-  description = "(Optional) The role that should be applied. Only one 'iam_binding' can be used per role. Note that custom roles must be of the format '[projects|organizations]/{parent-name}/roles/{role-name}'."
+  description = "(Required) The role that should be applied. Only one 'iam_binding' can be used per role. Note that custom roles must be of the format '[projects|organizations]/{parent-name}/roles/{role-name}'."
```

Comment:
role is not required by the module. please revert the change. in case policy bindings are used, role does not need to be set.